derive/src/compile_bytecode.rs

```diff
@@ -143,7 +143,11 @@ impl CompilationSource {
             if path.is_dir() {
                 code_map.extend(self.compile_dir(
                     &path,
-                    format!("{}{}", parent, file_name),
+                    if parent.is_empty() {
+                        format!("{}{}", parent, file_name)
```

Comment:
```suggestion
                        format!("{}", file_name)
```

Reply:
Yes. Though. Might as well just go with `file_name.to_string()` at that point. Will update the PR.